src/adapter/types.ts

```diff
@@ -99,17 +99,14 @@ export interface AdapterParams<CustomSettings extends SettingsMap> {
    */
   endpoints: AdapterEndpoint<any>[] // eslint-disable-line @typescript-eslint/no-explicit-any
 
-  /** Map of overrides to the default config values for an Adapter */
-  envDefaultOverrides?: Partial<BaseAdapterConfig>
-
-  /** List of custom env vars for this particular adapter (e.g. RPC_URL) */
-  customSettings?: SettingsMap
-
   /** Configuration relevant to outbound (EA --\> DP) communication rate limiting */
   rateLimiting?: AdapterRateLimitingConfig
 
   /** Bootstrap function that will run when initializing the adapter */
-  bootstrap?: (adapter: Adapter<CustomSettings>) => Promise<void>
+  bootstrap?: (adapter: Adapter<T>) => Promise<void>
+
+  /** TODO: complete */
```

Comment:
TODO - can you either finish this or add a ticket for it (if there isn't one already)?

Reply:
Completed